sites/authentication/authentication-en.html

```diff
@@ -0,0 +1,89 @@
+---
+{
+	"altLangPage": "authentication-fr.html",
+	"breadcrumb": {
+		"title": "Canada.ca", "link": "https://www.canada.ca/en.html"
+	},
+	"dateModified": "2022-05-04",
+	"description": "Documentation on how to use the various authentication patterns.",
+	"language": "en",
+	"title": "Authentication patterns"
+}
+---
+<p>The <code>auth</code> variable MUST be set along with the <code>type</code> parameter in order to initiate one of the authentication pattern.</p>
+
+<h2>Contextual Sign-in button</h2>
+<p>Setting the <code>type</code> parameter with the value "contextual" will initiate the contextual Sign-in button. With proper parameters, the button will appear on page load and will be located on the top right-hand side of the page, below the horizontal line as indicated in the <a href="https://design.canada.ca/common-design-patterns/buttons.html#sign-in">design.canada guidelines</a>.</p>
+
+<h3>Specifications</h3>
+
+<p>Parameters to be set:</p>
+<ul>
+	<li><code>type</code> this parameter MUST be set with the value "contextual" to initiate the contextual sign-in button pattern.</li>
+	<li><code>link</code> this parameter MUST be set in order to assign a URL to the link button for pointing to the application where users will be redirected.</li>
+	<li><code>label</code> with a maximum lenght of 12 characters allowed including spaces, this parameter MUST be set to generate the text appearing on the button for any viewports from extra-extra-small to extra-large (xxs, xxs, sm, md, lg and xl).</li>
+	<li><code>labelExtended</code> with a maximum lenght of 25 characters allowed including spaces, this parameter is optional but must be set along with the <code>label</code> paramter and if set, it will generate the text appearing on the button for small viewport and up (sm, md, lg and xl).</li>
+</ul>
```

Comment:
We need to separate Jekyll config from our monitored versioning API.

```suggestion
<p>With proper parameters, the button will appear on page load and will be located on the top right-hand side of the page, below the horizontal line as indicated in the <a href="https://design.canada.ca/common-design-patterns/buttons.html#sign-in">design.canada guidelines</a>.</p>

<h3>GCWeb Jekyll specifications</h3>
<p>Setting the <code>type</code> parameter with the value "contextual" will initiate the contextual Sign-in button.</p>
<p>Parameters to be set:</p>
<ul>
	<li><code>type</code> this parameter MUST be set with the value "contextual" to initiate the contextual sign-in button pattern.</li>
	<li><code>link</code> this parameter MUST be set in order to assign a URL to the link button for pointing to the application where users will be redirected.</li>
	<li><code>label</code> with a maximum lenght of 12 characters allowed including spaces, this parameter MUST be set to generate the text appearing on the button for any viewports from extra-extra-small to extra-large (xxs, xxs, sm, md, lg and xl).</li>
	<li><code>labelExtended</code> with a maximum lenght of 25 characters allowed including spaces, this parameter is optional but must be set along with the <code>label</code> paramter and if set, it will generate the text appearing on the button for small viewport and up (sm, md, lg and xl).</li>
</ul>

<h3>HTML Structure - Version 1.0</h3>
<pre><code>Code sample of the include showing the HTML of the component</code></pre>

```